x-pack/plugins/gis/public/shared/layers/sources/source.js

```diff
@@ -41,10 +37,6 @@ export class ASource {
     return '';
   }
 
-  isFilterByMapBounds() {
```

Comment:
Why was this removed from `ASource`?

Reply:
only vectorsources support filtering, it's not implemented for any other source.. also, VectorLayer is the only one using this method. removing it from ASource trims down the contract a little